docs/ProblemStatementAndGoals/ProblemStatement.tex

```diff
@@ -22,63 +22,99 @@
 \toprule
 \textbf{Date} & \textbf{Developer(s)} & \textbf{Change}\\
 \midrule
-Date1 & Name(s) & Description of changes\\
-Date2 & Name(s) & Description of changes\\
+09/18/2024 & Angela Wang & Initial Draft\\
 ... & ... & ...\\
 \bottomrule
 \end{tabularx}
 \end{table}
 
 \section{Problem Statement}
 
-\wss{You should check your problem statement with the
-\href{https://github.com/smiths/capTemplate/blob/main/docs/Checklists/ProbState-Checklist.pdf}
-{problem statement checklist}.} 
+% \wss{You should check your problem statement with the
+% \href{https://github.com/smiths/capTemplate/blob/main/docs/Checklists/ProbState-Checklist.pdf}
+% {problem statement checklist}.} 
 
-\wss{You can change the section headings, as long as you include the required
-information.}
+% \wss{You can change the section headings, as long as you include the required
+% information.}
 
 \subsection{Problem}
 
+Young adults often face challenges in managing their finances effectively,
+especially when it comes to tracking expenses and budgeting. While various tools are available, many rely on manual data entry, which is time-consuming and prone to errors. This inefficiency leads to frustration and creates a significant barrier for young adults seeking to gain control of their financial situation and develop better spending habits.
+
+
 \subsection{Inputs and Outputs}
 
-\wss{Characterize the problem in terms of ``high level'' inputs and outputs.  
-Use abstraction so that you can avoid details.}
+% \wss{Characterize the problem in terms of ``high level'' inputs and outputs.  
+% Use abstraction so that you can avoid details.}
+
+Inputs: User inputs data about their expenses and budget goals.
+
+Outputs: Visualizations of the user's spending allocations in comparison to their set budget, and recommendations for how they may adjust their spending to meet their goals.
 
 \subsection{Stakeholders}
 
+Stakeholders include anyone who is looking to better manage their finances, set budget goals, and track their spending, with a focus on young adults.
+
 \subsection{Environment}
 
 \wss{Hardware and software environment}
 
 \section{Goals}
 
+Our goals include the following:
+\begin{itemize}
+    \item Develop a machine learning model that can accurately (\textgreater90\%) parse commonly purchased items (e.g., groceries, cleaning supplies) from a picture of a receipt.
```

Comment:
oh also should we have a defined accuracy for this doc? or should we save until later @chene40 @paytonchan @fondsonlu FYI